lib/dialects/abstract/query-generator.js

```diff
@@ -101,7 +101,7 @@ const QueryGenerator = {
       emptyQuery += ' VALUES ()';
     }
 
-    if (this._dialect.supports.returnValues && options.returning) {
+    if (this._dialect.supports.returnValues && options.returning && this._dialect.name !== 'oracle') {
```

Comment:
This looks odd to me. Why does it need another check on whether the dialect name is 'oracle'? Shouldn't `this._dialect.supports.returnValues` be false already?

Also, not fully understanding the context here but there is `RETURNING` support in Oracle too: http://docs.oracle.com/database/122/LNPLS/RETURNING-INTO-clause.htm#LNPLS01354
But that could be something else than wanted here.

Reply:
needs investigation